jsonobject/containers.py

```diff
@@ -31,6 +31,10 @@ def __delitem__(self, i):
         super(JsonArray, self).__delitem__(i)
         del self._obj[i]
 
+    def __setitem__(self, index, wrapped):
+        self._obj[index] = wrapped
```

Comment:
@TylerSheffels This caused a bug, fixed https://github.com/dimagi/jsonobject/pull/122. Internal representations need to be unwrapped.